src/collections/_documentation/error-reporting/getting-started-verify/browser.md

```diff
@@ -1,3 +1,5 @@
+What better way to send your first event than to break your application!
+
 One way to break your JavaScript application is to call an undefined function:
```

Comment:
Maybe reword to "One way to do this (that?) is..." to not repeat "break your" twice?

Reply:
This repetition was already there for JS Browser, Cordova, JS Node, Electron. Now it just got obvious without rendering the docs :)


**CURRENT** https://docs.sentry.io/error-reporting/quickstart/?platform=browser#capturing-your-first-event
![image](https://user-images.githubusercontent.com/88819/75336176-f3f6f280-588a-11ea-95eb-1999549316ad.png)


Not too bad, is it? I'll reword it since we're already touching this section.